_includes/charts/tigera-operator/templates/tigera-operator/02-podsecuritypolicy-tigera-operator.yaml

```diff
@@ -0,0 +1,44 @@
+{{ if ne .Values.installation.kubernetesProvider "openshift" }}
```

Comment:
I don't think this if check is actually needed because to include the file for an openshift install it needs to exist in `manifests/ocp/tigera-operator/...` and included in `_includes/content/openshift-manfiests.md`.
If you do want to remove the check then I think you should add a comment that this file should not be used on an openshift install.

Reply:
As per our offline conversation, I added a comment and decided to keep this condition just to make sure that this is not used in OpenShift.